### PR TITLE
Follow focus while multiselect

### LINKED ISSUE #846: Focus during mutli-cell selection
Short description: selecting multiple cells can result in the active cell being off screen.

My environment:  
* Ubuntu + latest chrome/firefox
* Notebook v4.1.0b1

Reproducing:  
* Open a notebook with more cells than fits into a single screen
* Press shift+J (shift-K) several times

Result:  
Active cell is off screen

Expected behavior:  
Active cell is the lowest visible cell (highest visible cell for shift-K), same as with regular J/K.

### PR DESCRIPTION
**Pull request by Carreau**
Should fix #846, with a bunch of semicolon.

### CODE REVIEW

File: notebook/static/notebook/js/actions.js [hunk 1]
```diff
@@ -183,14 +183,16 @@ define(function(require){
             help: 'extend selected cells above',
             help_index : 'dc',
             handler : function (env) {
-                env.notebook.extend_selection_by(-1)
+                env.notebook.extend_selection_by(-1);
+                env.notebook.focus_cell();
```
Comment (Carreau) [inline notebook/static/notebook/js/actions.js]: relevant line 1/3

File: notebook/static/notebook/js/actions.js [hunk 2]
```diff
@@ -183,14 +183,16 @@ define(function(require){
...
             }
         },
         'extend-selection-below' : {
             help: 'extend selected cells below',
             help_index : 'dd',
             handler : function (env) {
-                env.notebook.extend_selection_by(1)
+                env.notebook.extend_selection_by(1);
+                env.notebook.focus_cell();
```
Comment (Carreau) [inline notebook/static/notebook/js/actions.js]: relevant line 2/3

File: notebook/static/notebook/js/notebook.js [hunk 3]
```diff
@@ -607,7 +607,7 @@ define(function (require) {
      * @return {Cell} The selected cell
      */
     Notebook.prototype.get_selected_cell = function () {
-        var index = this.get_selected_cells_indices();
+        var index = this.get_selected_index();
```
Comment (Carreau) [inline notebook/static/notebook/js/notebook.js]: relevant line 3/3

**Comment by minrk** [comment]
Tested and confirmed the fix locally.

**Pull request merged**